web/translation/translate.fa_IR.toml

```diff
@@ -213,151 +213,151 @@
-"IRIp" = "جلوگیری از اتصال آیپی های ایران"
-"IRIpDesc" = "الگوی تنظیمات را برای فیلتر کردن اتصال آیپی های ایران تغییر میدهد"
-"IRDomain" = "جلوگیری از اتصال دامنه های ایران"
-"IRDomainDesc" = "الگوی تنظیمات را برای فیلتر کردن اتصال دامنه های ایران تغییر میدهد"
-"ChinaIp" = "جلوگیری از اتصال آیپی های چین"
-"ChinaIpDesc" = "الگوی تنظیمات را برای فیلتر کردن اتصال آیپی های چین تغییر میدهد"
-"ChinaDomain" = "جلوگیری از اتصال دامنه های چین"
-"ChinaDomainDesc" = "الگوی تنظیمات را برای فیلتر کردن اتصال دامنه های چین تغییر میدهد"
-"RussiaIp" = "جلوگیری از اتصال آیپی های روسیه"
-"RussiaIpDesc" = "الگوی تنظیمات را برای فیلتر کردن اتصال آیپی های روسیه تغییر میدهد"
-"RussiaDomain" = "جلوگیری از اتصال دامنه های روسیه"
-"RussiaDomainDesc" = "الگوی تنظیمات را برای فیلتر کردن اتصال دامنه های روسیه تغییر میدهد"
-"DirectIRIp" = "ارتباط مستقیم به آیپی های ایران"
-"DirectIRIpDesc" = "الگوی تنظیمات را برای ارتباط مستقیم به آیپی های ایران تغییر میدهد"
-"DirectIRDomain" = "ارتباط مستقیم به دامنه های ایران"
-"DirectIRDomainDesc" = "الگوی تنظیمات را برای ارتباط مستقیم به دامنه های ایران تغییر میدهد"
-"DirectChinaIp" = "ارتباط مستقیم به آیپی های چین"
-"DirectChinaIpDesc" = "الگوی تنظیمات را برای ارتباط مستقیم به آیپی های چین تغییر میدهد"
-"DirectChinaDomain" = "ارتباط مستقیم به دامنه های چین"
-"DirectChinaDomainDesc" = "الگوی تنظیمات را برای ارتباط مستقیم به دامنه های چین تغییر میدهد"
-"DirectRussiaIp" = "ارتباط مستقیم به آیپی های روسیه"
-"DirectRussiaIpDesc" = "الگوی تنظیمات را برای ارتباط مستقیم به آیپی های روسیه تغییر میدهد"
-"DirectRussiaDomain" = "ارتباط مستقیم به دامنه های روسیه"
-"DirectRussiaDomainDesc" = "الگوی تنظیمات را برای ارتباط مستقیم به دامنه های روسیه تغییر میدهد"
-"GoogleIPv4" = "استفاده از آیپی ورژن 4 برای اتصال به گوگل"
-"GoogleIPv4Desc" = "مسیردهی جدید برای اتصال به گوگل با آیپی ورژن 4 اضافه میکند"
-"NetflixIPv4" = "استفاده از آیپی ورژن 4 برای اتصال به نتفلیکس"
-"NetflixIPv4Desc" = "مسیردهی جدید برای اتصال به نتفلیکس با آیپی ورژن 4 اضافه میکند"
+"basicTemplate" = "الگو پایه"
+"advancedTemplate" = "الگو پیشرفته"
+"generalConfigs" = "استراتژی مسیریابی"
+"generalConfigsDesc" = "این گزینه‌هااستراتژی کلی مسیریابی ترافیک را بازتاب می‌دهد"
+"blockConfigs" = "مسدودسازی پایه" 
+"blockConfigsDesc" = "این گزینه‌ها ترافیک را بر اساس پروتکل‌های درخواستی خاص، و وب سایت ها مسدود می‌کند"
+"blockCountryConfigs" = "مسدودسازی کشور"
+"blockCountryConfigsDesc" = "این گزینه‌ها ترافیک را بر اساس کشور درخواستی خاص مسدود می‌کند"
+"directCountryConfigs" = "اتصال مستقیم کشور"
+"directCountryConfigsDesc" = "این گزینه‌ها ترافیک را بر اساس کشور درخواستی خاص بصورت مستقیم ارسال می‌کند"
+"ipv4Configs" = "IPv4 مسیریابی"
+"ipv4ConfigsDesc" = "این گزینه‌ها درخواست ها را فقط از طریق آی‌پی نسخه4 به مقصد هدایت می‌کند"
+"Template" = "‌پیکربندی پیشرفته الگو ایکس‌ری"
+"TemplateDesc" = "فایل پیکربندی نهایی ایکس‌ری بر اساس این الگو ایجاد می‌شود"
+"FreedomStrategy" = "Freedom استراتژی مسیریابی برای پروتکل"
+"FreedomStrategyDesc" = "Freedom تعیین استراتژی استفاده از خروجی برای پروتکل"

```

Comment:
It is not about Routing strategy. Freedom is an outbound protocol.
This option will configure a setting for this outbound.